group_manager/group_manager.py

```diff
@@ -21,8 +21,12 @@ def index() -> Response:
     user_type = response['data']['user_type']
     user_zone = response['data']['user_zone']
 
+    response = api.call('get_schemas', data={})
```

Comment:
Is this API call still needed? Looks like the schemas are retrieved by select2.

Reply:
This is the initial list for schema-validation purposes when a new group is added.
This list could be derived from the select2 but this select2 is not nessecarily used at that point.